test/e2e/mixedcpus/mixedcpus.go

```diff
@@ -47,20 +48,20 @@ const (
 )
 
 var _ = Describe("Mixedcpus", func() {
+	fxt := fixture.New()
```

Comment:
a fixture per `Describe` block is probably the best compromise. Ideally each `It` block should have its own